docs/src/languages/r.md

```diff
@@ -59,7 +48,56 @@ exclusions: list(".")
 
 See [Using lintr](https://lintr.r-lib.org/articles/lintr.html) for a complete list of options,
 
-### Formatting
+## Formatting
+
+### Air
+
+[Air](https://posit-dev.github.io/air/) provides code formatting for R, including support for format-on-save. The [Air documentation for Zed](https://posit-dev.github.io/air/editor-zed.html) contains the most up-to-date advice for running Air in Zed.
+
+Ensure that you have installed both the [ocsmit/zed-r](https://github.com/ocsmit/zed-r) extension (for general R language awareness in Zed) and the [Air](https://posit-dev.github.io/air/) extension.
```

Comment:
I once again wish there was a better way for basic R language support (literally just recognizing that it's an R file, and tree-sitter support) to be shipped in Zed in a way that was decoupled from the {languageserver} that comes with zed-r!

https://github.com/zed-industries/zed/discussions/25758